src/styles/calendar.scss

```diff
@@ -0,0 +1,20 @@
+.calendar {
+  gap: $daysGaps;
+  padding: 20px;
+
+  &__day {
+    @include dayBlockSize();
+
+    border: 1px solid black;
+    background-color: #eee;
+    text-align: center;
+    line-height: $dayBlockHeight;
+  }
+}
+
+.calendar__day:hover {
```

Comment:
[SASS] - Make use of SASS nesting - write pseudo-class, pseudo-element selectors inside general selector. As well as media queries.